ui/src/virtualconsole/vccuelist.cpp

```diff
@@ -676,6 +676,8 @@ void VCCueList::slotStop()
                                             .arg(m_stopButton->palette().window().color().name()));
             m_progress->setFormat("");
             m_progress->setValue(0);
+
+            emit slotCueProgressState();
```

Comment:
Signals are not prefixed with the "slot" keyword (slots are)
Please rename to "progressStateChanged"